spec/models/order_import_spec.rb

```diff
@@ -1,342 +1,374 @@
-      :name => "Example Service"
-    ))
+  before(:all) { create_users }
 
-    # price stuff
-    @price_group      = @authable.price_groups.create!(FactoryGirl.attributes_for(:price_group))
-    @pg_member        = FactoryGirl.create(:user_price_group_member, :user => @guest, :price_group => @price_group)
-    @item_pp=@item.item_price_policies.create!(FactoryGirl.attributes_for(:item_price_policy,
-      :price_group_id => @price_group.id,
-      :start_date => fiscal_year_beginning
+  before :each do
+    grant_role(director, facility)
 
+    price_group = facility.price_groups.create!(attributes_for(:price_group))
+    create(:user_price_group_member, user: guest, price_group: price_group)
+    item.item_price_policies.create!(attributes_for(:item_price_policy,
+      price_group_id: price_group.id,
+      start_date: fiscal_year_beginning,
     ))
-    @service_pp=@service.service_price_policies.create!(FactoryGirl.attributes_for(:service_price_policy,
-      :price_group_id => @price_group.id,
-      :start_date => fiscal_year_beginning
+    service.service_price_policies.create!(attributes_for(:service_price_policy,
+      price_group_id: price_group.id,
+      start_date: fiscal_year_beginning,
     ))
 
-    @guest2 = FactoryGirl.create :user, :username => 'guest2'
-    @pg_member        = FactoryGirl.create(:user_price_group_member, :user => @guest2, :price_group => @price_group)
-    @account          = FactoryGirl.create(:nufs_account,
-      :description => "dummy account",
-      :account_number => '111-2222222-33333333-01',
-      :account_users_attributes =>
-        (account_users_attributes_hash(:user => @guest) +
-         account_users_attributes_hash(:user => @guest2, :created_by => @guest, :user_role => AccountUser::ACCOUNT_PURCHASER))
-    )
-
+    create(:user_price_group_member, user: guest2, price_group: price_group)
 
-    stored_file = StoredFile.create!(
-      :file => StringIO.new("c,s,v"),
-      :file_type => 'import_upload',
-      :name => "clean_import.csv",
-      :created_by => @director.id
+    create(:nufs_account,
+      description: "dummy account",
+      account_number: '111-2222222-33333333-01',
+      account_users_attributes: account_users_attributes,
     )
+  end
 
-    @order_import=OrderImport.create!(
-      :created_by => @director.id,
-      :upload_file => stored_file,
-      :facility => @authable
-    )
+  shared_examples_for "it does not send notifications" do
+    it "does not send notifications" do
+      expect(Notifier).to receive(:order_receipt).never
+      order_import.process_upload!
+    end
   end
 
-  after :each do
-    Timecop.return
+  context "validations" do
+    it { should belong_to :creator }
+    it { should belong_to :upload_file }
+    it { should belong_to :error_file }
+    it { should validate_presence_of :upload_file }
+    it { should validate_presence_of :created_by }
   end
 
+  describe "#import_row (low-level behavior)" do
```

Comment:
Since we have separate specs for `OrderImportRow`, do we need all these specs here now?